src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Consider using the checklist here to refactor your code https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md